### ISSUE TITLE
beatoraja crash when scrolling over song with no #BPM field

### ISSUE THREAD

**Issue by wcko87** [exch-bms2/beatoraja#343]
If a bms has no `#BPM` field, scrolling over the song in music select will crash beatoraja.

This crash happens with a PMS chart in GENOSIDE 2018 (DataErr0r, 9KEY Hyper)
[dataerr0r_9h_pms.zip](https://github.com/exch-bms2/beatoraja/files/2184661/dataerr0r_9h_pms.zip)

If you add `#BPM 180` to the file, beatoraja will not crash.

Crash video: [VIDEO](https://gfycat.com/AnxiousCoordinatedLarva)

```
SEVERE: BMS Search??????:Field not found: id (bms.player.beatoraja.external.BMSSearchAccessor$Fumen)
Serialization trace:
[][0].fumen[0].id
fumen (bms.player.beatoraja.external.BMSSearchAccessor$BMSSearchElement)
Exception in thread "LWJGL Application" java.lang.NegativeArraySizeException
        at bms.player.beatoraja.skin.SkinNoteDistributionGraph.updateGraph(Unknown Source)
        at bms.player.beatoraja.skin.SkinNoteDistributionGraph.draw(Unknown Source)
        at bms.player.beatoraja.skin.SkinNoteDistributionGraph.draw(Unknown Source)
        at bms.player.beatoraja.skin.Skin.drawAllObjects(Unknown Source)
        at bms.player.beatoraja.MainController.render(Unknown Source)
        at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:225)
        at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126)
```

**Comment by exch-bms2** [comment]
This PMS chart lacks BPM definition in the section #000-#007. Maybe some BMS players also crash when parsing it.,so that I think beatoraja needs to reject such a illegal chart. What do you think?

**Comment by wcko87** [comment]
Yes I think beatoraja should ignore invalid charts like this, or display an error message that the chart is invalid and cannot be played. Crashing suddenly is confusing to the user.

The same exception also occurs when the song pack is added in the config menu.

**Comment by wcko87** [comment]
It seems like according to the (very old) bms spec, default bpm is 130 if not specified.
http://bm98.yaneu.com/bm98/bmsformat.html

I think LR2 uses the first bpm change if no bpm is specified.

**Comment by exch-bms2** [comment]
fixed on 0.6.1. (ingores start bpm = 0 chart)
